src/www/app/outline_server_repository/index.ts

```diff
@@ -144,13 +154,29 @@ export class OutlineServerRepository implements ServerRepository {
   }
 
   validateAccessKey(accessKey: string) {
-    const alreadyAddedServer = this.serverFromAccessKey(accessKey);
+    if (!this.isDynamicAccessKey(accessKey)) {
+      return this.validateStaticKey(accessKey);
+    }
+
+    try {
+      new URL(accessKey);
+    } catch (error) {
+      throw new errors.ServerUrlInvalid(error.message);
+    }
+  }
+
+  private isDynamicAccessKey(accessKey: string) {
+    return !accessKey.startsWith('ss://');
```

Comment:
Is this the proper criteria for determining if a key is dynamic?

Reply:
Perhaps it's clearer if we use `ssconf`, though it's fine to accept https too  just in case:
```suggestion
    return accessKey.startsWith('ssconf://') || accessKey.startsWith('https://');
```